src/amber.cr

```diff
@@ -18,8 +44,8 @@ module Amber
       instance.all_routes
     end
 
-    def self.settings
-      instance
+    def self.config(&block)
```

Comment:
🥇 I really like this.  Should we should consider this pattern for the pipes as well?  I had something similar in Kemalyst.